src/main/java/com/rultor/profiles/Profiles.java

```diff
@@ -73,26 +72,37 @@ public Profile fetch(final Talk talk) throws IOException {
             if (xml.nodes("/talk/wire").isEmpty()) {
                 profile = Profile.EMPTY;
             } else {
-                profile = Profiles.fetch(xml);
+                profile = this.fetch(xml);
             }
         }
         return profile;
     }
 
     /**
+     * Merge profile from master and fork.
```

Comment:
@xupyprmv can we have more info in Javadoc describing purpose and details of merging? I'm ok to add this task to todo